test/apps/iodemo/io_demo.cc

```diff
@@ -1380,10 +1380,10 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
     int c;
 
     test_opts->port_num              = 1337;
-    test_opts->connect_timeout       = 20.0;
-    test_opts->client_timeout        = 50.0;
+    test_opts->connect_timeout       = 10.0;
+    test_opts->client_timeout        = 10.0;
     test_opts->client_retries        = std::numeric_limits<long>::max();
-    test_opts->client_retry_interval = 5.0;
```

Comment:
can we avoid changing these?

Reply:
we can try, but there will be more risks for io_demo job to fail. If network corrupter will bring down the port while not all traffic was sent yet, it may take a while to recover (and we do not have keepalive here). And the test job can fail.
To avoid that, I can reduce iterations number even more (say 100000) for now, but then there will be no traffic to recover in most of the cases, as it will be sent before port shutdown.

Reply:
let's reduce # of iterations then, there are reasons to keep the intervals in same values as int3, i'd rather reduce the differences between master and int3